### PR TITLE
Make some Jenkins Tests

### PR DESCRIPTION
**Pull request by VasilyiD**
My first jenkins tests

### CODE REVIEW

File: src/test/java/school/redrover/VasilyiDrakeTest.java [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    public void createFreestyleProjectDuplicateName() {
+
+        getProjectPage();
+        getDriver().findElement(By.xpath("//a[contains(text(), 'Dashboard')]")).click();
+        getDriver().findElement(By.xpath("//a[contains(@href, 'newJob')]")).click();
+        getDriver().findElement(By.xpath("//*[starts-with(@name,'name') and contains(@id,'name')]")).sendKeys(PROJECT_NAME);
+        getDriver().findElement(By.id("ok-button")).click();
+
+        Assert.assertEquals(getDriver().findElement(By.xpath("//*[@class = 'input-validation-message']")).getText(),
+                "» A job already exists with the name ‘Starlight’");
```
Comment (bzhumanova) [inline src/test/java/school/redrover/VasilyiDrakeTest.java:68]: do you think it makes sense to make it like this "...with the name " +. PROJECT_NAME)

File: src/test/java/school/redrover/VasilyiDrakeTest.java [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+        Assert.assertEquals(getDriver().findElement(By.xpath("//h1[text()='Welcome to Jenkins!']")).getText(),
+                            "Welcome to Jenkins!");
+
+    }
+
+
+    }
+
+
```
Comment (bzhumanova) [inline src/test/java/school/redrover/VasilyiDrakeTest.java:88]: please remove empty lines, thanks

**Review by bzhumanova — APPROVED** [review]

File: src/test/java/school/redrover/VasilyiDrakeTest.java [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+
+public class VasilyiDrakeTest  extends BaseTest {
+
+
+    private static final String PROJECT_NAME = "Starlight";
+
+    private void getNewJobPage() {
```
Comment (svcat) [inline src/test/java/school/redrover/VasilyiDrakeTest.java:14]: 👍 Good not to repeat yourself

**Review by svcat — APPROVED** [review]

File: src/test/java/school/redrover/VasilyiDrakeTest.java (hunk 1 above)
Comment (VasilyiD) [reply src/test/java/school/redrover/VasilyiDrakeTest.java:68]: I thought that if I use the value more than 1 time, it's better to make a variable

**Pull request merged**